# HG changeset patch
# User Rik <rik@octave.org>
# Date 1470786637 25200
#      Tue Aug 09 16:50:37 2016 -0700
# Node ID a4ff69a4f2398658f6452d80a98604ca8b16e1ff
# Parent  654de580bdb38d76a8fe1a016c45395e8c271c74
__unimplemented__.m: Don't print backtrace when issuing warning.

* __unimplemented__.m: Add "\n" to call to warning to stop backtrace.

diff --git a/scripts/help/__unimplemented__.m b/scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m
+++ b/scripts/help/__unimplemented__.m
@@ -510,17 +510,17 @@ function txt = __unimplemented__ (fcn)
   if (is_matlab_function)
     txt = [txt, "\n\n@noindent\nPlease read ", ...
            "@url{http://www.octave.org/missing.html} to learn how ", ...
            "you can contribute missing functionality."];
     txt = __makeinfo__ (txt);
   endif
 
   if (nargout == 0)
-    warning ("Octave:missing-function", "%s", txt);
+    warning ("Octave:missing-function", "%s\n", txt);
   endif
 
 endfunction
 
 function txt = check_package (fcn, name)
 
   txt = sprintf ("the '%s' function belongs to the %s package from Octave Forge",
                  fcn, name);
